spec/extracted_code_spec.rb

```diff
@@ -0,0 +1,121 @@
+describe "Files and directories that shouldn't exist" do
+  let(:all_paths) { Dir.chdir(Rails.root) { Dir.glob("**/*") } }
```

Comment:
`Dir.glob(Rails.root.join("**/*"))`

I don't think you need the chdir

Reply:
I did the chdir to get the path relative to the root of the repo.  If I don't, I'll get `/home/bdunne/.....`

Reply:
Ah, I thought it was only being used for the basenames, but I see you use it directly in the test, so :+1: